server/APIOptions/twitterOptions.js

```diff
@@ -12,9 +12,9 @@ options: { method: 'GET',
      screen_name: 'jpackel',
      count: '20' },
   headers:
-   {'access-token': process.env.twitter_access_token,
+   {'access-token': process.env.twitter_access_token, // dotenv not working, so we'll module exports in the .env file for now
```

Comment:
please add the needed .env require statement and proper reference to make this work.